sources/javascripts/concat/base.js

```diff
@@ -311,7 +311,7 @@
         Edicy.articles.currentArticle.setData(dataBgKey, commitData);
       }
     } else {
-      if (pageType === 'contentPage' && (dataBgKey === 'footer_bg') || (dataBgKey === 'body_bg')) {
+      if (pageType === 'contentPage' && (dataBgKey === 'footer_bg') || (dataBgKey === 'body_bg') || dataBgKey === 'anchorage_product')  {
```

Comment:
`'anchorage_product'` seems too specific in JS file.

I'll recommend to introduce `pageType === 'productPage'` or `pageType === 'site'` value for `pageType` in case of product page (see fornt page pattern: https://github.com/Voog/design-anchorage/blob/a0a779549d86457ab2ca6354424dc448b2808f72/layouts/front_page.tpl#L2 and https://github.com/Voog/design-anchorage/blob/69ff48a11ac2a4b2574aca47bf988a315615a98b/components/template-tools.tpl#L19)